code/game/objects/items/weapons/kitchen.dm

```diff
@@ -98,6 +98,14 @@
 	origin_tech = "materials=2;combat=4"
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "cut")
 
+/obj/item/weapon/kitchen/knife/carrotshiv

```

Comment:
However, if you make them a child of screwdriver, you could stab peoples eyes out and unscrew windows with it.